crypto/rand/drbg_lib.c

```diff
@@ -501,31 +630,42 @@ void *RAND_DRBG_get_ex_data(const RAND_DRBG *drbg, int idx)
  */
 
 /*
- * Initializes the DRBG with default settings.
+ * Initializes the given global DRBG with default settings.
  * For global DRBGs a global lock is created with the given name
```

Comment:
I think it's now only used for global DRBGs and the lock is always created, so this comment seems out of date.

Reply:
Canged to 

```
 * Initializes the given global DRBG with default settings.
 * A global lock for the DRBG is created with the given name.
```